tests/integ/modin/io/test_to_csv.py

```diff
@@ -256,3 +256,36 @@ def test_to_csv_unsupported_params_error(sf_stage, session, kwargs):
     with pytest.raises(NotImplementedError, match=msg):
         # Write csv to snowflake stage.
         pd.DataFrame(native_df).to_csv(stage_location, **kwargs)
+
+
+def test_to_csv_timedelta_local():
+    kwargs = {"columns": ["B"]}
+    native_df = native_pd.DataFrame({"A": ["one", "two", None], "B": [1, 2, 3]})
+    native_path, snow_path = get_filepaths(kwargs, "dataframe_local")
+
+    kwargs = kwargs.copy()
+    kwargs.pop("ext", None)
```

Comment:
```suggestion
    kwargs = kwargs.copy()
    kwargs.pop("ext", None)
```

I don't think we need this